### ISSUE TITLE
Possible problem with the update of the tags in the FORMAT field

### ISSUE THREAD

**Issue by everestial** [secastel/phaser#24]
Hi Castel,

I raised this issue before but I think I had lots of other issues in that section and might have been buried. The problem is with how the tags and it's values get updated in the FORMAT field as you run phASER on one sample first and again on that output vcf to phase the another sample.

Say I have a vcf file with several samples:  samole1, sample2, sample3
And, if these were the tags in the FORMAT field: GT:AD:DP:GQ:PL
I run the phaser on `sample1` which will now update the `FORMAT` field to `GT:AD:DP:GQ:PB:PC:PG:PI:PL:PW	`

Since, I have to run phASER on another sample now, I take the output from the previous run and now the `FORMAT` fields are updated as:  `GT:AD:DP:GQ:PB:PC:PG:PI:PL:PW:PB:PC:PG:PI:PW` leading to two tags and two values (in some cases) for a sample.

I am still using the old pHASER due to issues with Cython, but wanted to highlight if this fix is done (if not a priority) in updated phASER versions.

Thanks,

**Comment by secastel** [comment]
Yes, I seem to have missed this issue - I apologize. I have fixed this in the most recent version of phASER (v0.9.9.3) which I just pushed to github. If a VCF that has already been run through phASER is used it will now properly use the previous formatting instead of adding new fields.

Aside from this, there has been a slight change in the way phASER handles multisample VCFs that was implemented a while back. As of newer versions, when a multisample VCF is used, phASER will only output the sample that it was run on in the output VCF. The resulting per sample VCFs can then be combined extremely efficiently using [bcftools](https://samtools.github.io/bcftools/bcftools.html) merge. We found this was the most efficient solution because it allows large multisample projects to be run in parallel and then merged, as opposed to having to do samples sequentially.

Please let me know if this answers your question and solves your problem.

Best,
Stephane

**Comment by everestial** [comment]
ok, thanks !
